compiler/x/codegen/OMRInstOpCode.cpp

```diff
@@ -86,7 +86,11 @@ template <typename TBuffer> typename TBuffer::cursor_t OMR::X86::InstOpCode::OpC
 
    if (encoding == OMR::X86::Default)
       {
-      enc = comp->target().cpu.supportsAVX() ? vex_l : OMR::X86::Legacy;
+#if !defined(J9VM_OPT_JITSERVER) && !defined(J9VM_OPT_CRIU_SUPPORT)
+      static
+#endif
+      bool avxSupport = comp->target().cpu.supportsAVX();
```

Comment:
The main issue here is that these macros are OpenJ9 specific, so it's going to pollute the OMR code unnecessarily. Additionally, even outside of JITServer and CRIU, `comp->target()` can be different in a normal and relocatable compilation.

I think some of the answers to these queries should be cached in the codegen object, so that it is only initialized once per compilation.